llvm/test/CodeGen/X86/pr86305.ll

```diff
@@ -0,0 +1,74 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -mtriple=x86_64-linux-gnu -mattr=avx512bf16 | FileCheck %s
```

Comment:
worth adding `-mattr=avx512bf16,-mattr=avx512vl` and `-mattr=+avxneconvert` RUN lines as well?

Reply:
The tests were copied from bfloat.ll, so they have already tested.